virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -286,6 +283,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //4.14 Leave in per Simone
 * Uninstalling {VirtProductName} does not remove the `feature.node.kubevirt.io` node labels created by {VirtProductName}. You must remove the labels manually. (link:https://issues.redhat.com/browse/CNV-22036[*CNV-22036*])
 
+//BZ 2151169
+* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV Reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
```

Comment:
```suggestion
* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
```
Lowercase according to [link](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/tlfs/datatypes/hv_partition_privilege_mask#:~:text=of%20guest%20debugging.-,AccessReenlightenmentControls,The%20partition%20has%20access%20to%20reenlightenment%20controls.,-CreatePartitions).

Reply:
Good catch, fixed now